jobs/settings/__init__.py

```diff
@@ -105,7 +108,7 @@
 PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
 
 STATIC_ROOT = os.path.join(PROJECT_ROOT, "staticfiles")
-# ALLOWED_HOSTS = ['django-portal.herokuapp.com', 'localhost', 'jobs.manjurulhoque.com', '127.0.0.1', 'localhost:3000']
+# ALLOWED_HOSTS = ['django-portal.herokuapp.com', .backends.database'localhost', 'jobs.manjurulhoque.com', '127.0.0.1', 'localhost:3000']
```

Comment:
Hi @wolframtheta, '.backends.database'  looks like a mistake here.

Reply:
Yes @wolframtheta it appear a mistake and this line should revert.

Reply:
Didn't realize about that mistake because it's in a comment :laughing: